src/cbuild/wrappers/cargo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo "Wrapper called with $@" > cargo-wrapper-invocation-$(date -Iseconds).log
+
+if [ -n "$CBUILD_NO_CARGO_AUDITABLE" ]; then
+    exec /usr/bin/cargo "$@"
+elif command -v cargo-auditable>/dev/null; then
+    exec /usr/bin/cargo auditable "$@"
```

Comment:
```suggestion
if [ -n "$CBUILD_NO_CARGO_AUDITABLE" ]; then
    exec /usr/bin/cargo "$@"
elif command -v cargo-auditable >/dev/null; then
    exec /usr/bin/cargo auditable "$@"
```

Reply:
eh, had that in for debugging purposes and forgot to remove it. Thanks!